server/routes/payment.js

```diff
@@ -37,35 +38,36 @@ router.get('/', (req, res) => {
     res.send("Welcome to payment route")
 })
 
-router.post('/payment', (req, res) => {
+router.post('/payment/:amount', (req, res) => {
     let create_payment_json = {
         "intent": "sale",
         "payer": {
             "payment_method": "paypal"
         },
         "redirect_urls": {
+            //Redirect to next page
             "return_url": "http://localhost:5000/pay/success",
```

Comment:
Make a TODO here to change the link later because this link will likely not work in deployment.